### PR TITLE
Added a LaunchScreen

### PR DESCRIPTION
**Pull request by kiasha10**
[Feature/Merge] - LaunchScreen and changed app icon on Login screen 

 [Merge] Feature/launchScreen → dev
 [Feature] - LaunchScreen

Simulator Screenshot - iPhone 15 Pro - 2024-05-23 at 13 34 27

 ## Description
Added launchScreen

## Checklist
Please review and check the following before submitting this pull request:
- [ ] The code follows the project's coding standards.
- [ ] 100% Unit test coverage for View Models

### CODE REVIEW

File: BlitzBoard/Features/LoginScreen.storyboard [hunk 1]
```diff
@@ -147,7 +148,7 @@
         </scene>
     </scenes>
     <resources>
-        <image name="LoginScreenIcon" width="564" height="564"/>
+        <image name="LaunchScreen" width="564" height="564"/>
```
Comment (kieran-woodrow) [inline BlitzBoard/Features/LoginScreen.storyboard]: Just checking, Is this correct that you want Login Screen to have Launch screen image? If so, the naming is just confusing then :)

**Review by kieran-woodrow — CHANGES_REQUESTED** [review]

File: BlitzBoard/Features/LoginScreen.storyboard (hunk 1 above)
Comment (kiasha10) [reply BlitzBoard/Features/LoginScreen.storyboard]: okay ill change it

**Review by kieran-woodrow — APPROVED** [review]

**Review by Kaityss — APPROVED** [review]
Nice Job, liking the logo

**Review by ksookdhew — APPROVED** [review]
Nice Job!

**Pull request merged**